sdk/search/azure-search-documents/CHANGELOG.md

```diff
@@ -1,15 +1,26 @@
 # Release History
 
-## 11.6.0-beta.10 (Unreleased)
+## 11.6.0-beta.10 (2023-10-12)
 
 ### Features Added
 
+- `SemanticQuery` has been added to `SearchOptions` to support semantic search.
```

Comment:
This one is a bit more nuanced as this isn't the Semantic search that existed before but a secondary semantic search.